e2e/wdio/platform/tests/input.e2e-spec.ts

```diff
@@ -0,0 +1,158 @@
+        webDriver.waitElementToBePresentInDOM(inputPage.textInput);
+        webDriver.waitForDisplayed(inputPage.textInput);
+        webDriver.addValue(inputPage.textInput, inputTestData.number);
+        webDriver.addValue(inputPage.textInput, inputTestData.special_characters);
+        webDriver.addValue(inputPage.textInput, inputTestData.text);
+
+        expect(webDriver.getValue(inputPage.textInput))
+            .toEqual(inputTestData.number + inputTestData.special_characters + inputTestData.text); // ???
+    });
+    // TODO: it is not working the same for manual and automation.
+    xit('impose any filters on the kind of input values the component receives (number)', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.numberInput);
+        webDriver.waitForDisplayed(inputPage.numberInput);
+        webDriver.click(inputPage.numberInput);
+
+        webDriver.sendKeys(inputTestData.number);
+        webDriver.sendKeys(inputTestData.special_characters);
+        webDriver.sendKeys(inputTestData.text);
+
+        expect(webDriver.getText(inputPage.numberInput)).toEqual(inputTestData.number);
+    });
+
+    it('wrap the input characters to the next line', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.defaultInput);
+        webDriver.waitForDisplayed(inputPage.defaultInput);
+        const heightBefore = webDriver.getElementSize(inputPage.defaultInput, 'height') as number;
+        webDriver.setValue(inputPage.defaultInput, inputTestData.longLine);
+        const heightAfter = webDriver.getElementSize(inputPage.defaultInput, 'height') as number;
+
+        expect(heightBefore).toBeLessThanOrEqual(heightAfter);
+    });
+
+    it('enable editing the entered characters', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.defaultInput);
+        webDriver.waitForDisplayed(inputPage.defaultInput);
+        webDriver.setValue(inputPage.defaultInput, inputTestData.text);
+        webDriver.sendKeys('Backspace');
+
+        expect(webDriver.getValue(inputPage.defaultInput)).toBe(inputTestData.text.slice(0, -1));
+        webDriver.clearValue(inputPage.defaultInput);
+        expect(webDriver.getValue(inputPage.defaultInput)).toBe('');
+    });
+
+    it('check have disabled attr assigned', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.disabledInput);
+        webDriver.waitForDisplayed(inputPage.disabledInput);
+
+        expect(webDriver.getAttributeByName(inputPage.disabledInput, 'ng-reflect-is-disabled')).toBe('true');
+        expect(webDriver.isEnabled(inputPage.disabledInput)).toBe(false);
+    });
+
+    it('have placeholder', () => {
+        expect(webDriver.getAttributeByNameArr(inputPage.inputsArray, 'placeholder', 1))
+            .toEqual(inputPContent.placeholdersArray);
+    });
+
+    it('should have error border color', () => {
+        if (browser.capabilities.browserName === 'Safari' || browser.capabilities.browserName === 'internet explorer') {
```

Comment:
I think we can update these with the new methods: webdriver.browerIsIEorSafari()